bindings/c/include/opendal.h

```diff
@@ -809,8 +809,8 @@ struct opendal_result_stat opendal_operator_stat(const struct opendal_operator_p
  *
  * * If the `path` points to NULL, this function panics, i.e. exits with information
  */
-struct opendal_result_list opendal_operator_blocking_list(const struct opendal_operator_ptr *ptr,
-                                                          const char *path);
+struct opendal_result_list opendal_operator_list(const struct opendal_operator_ptr *ptr,
```

Comment:
Please note that our core API has been changed:

- list() returns a list of entries.
- lister() returns a lister.

So this API should be renamed to `opendal_operator_lister` and should add a new API called `opendal_operator_list`.

Reply:
I will address this later on